src/background_script.js

```diff
@@ -0,0 +1,31 @@
+const metaAssets = '../meta_assets/';
+var selectedId = -1;
+
+function getIconPath(response) {
+	const sufix = response.length ? '' : `_without`;
```

Comment:
Im getting an error in chrome on this line:

![image](https://user-images.githubusercontent.com/17054057/89292083-06583200-d65c-11ea-9b88-ed46dd2266f7.png)

Reply:
also please rename to `suffix` instead of `sufix` 😄

Reply:
Fixed, sometimes the responseCallback provide an undefined. I set a default empty array.